deploy/complete/docker-compose/docker-compose.yml

```diff
@@ -18,14 +18,16 @@ services:
 
   # Storefront application middleware
   api:
-    image: iad.ocir.io/oracle/ateam/mushop-api:2.0.0
+    image: iad.ocir.io/oracle/ateam/mushop-api:2.0.1
     hostname: api
     restart: always
     cap_drop:
       - all
     cap_add:
       - NET_BIND_SERVICE
     read_only: true
+    ports:
```

Comment:
Exposing ports here to have the ability to directly hit the API without the edge router.